pkg/operator/encryption/shared_encryption.go

```diff
@@ -126,6 +126,15 @@ const encryptionSecretMigrationInterval = 30 * time.Minute // TODO how often?  -
 // (i.e. for AES-CBC mode the value associated with this map key is 32 bytes of random noise).
 const encryptionSecretKeyData = "encryption.operator.openshift.io-key"
 
+// These annotations try to scare anyone away from editing the encryption secrets.  It is trivial for
+// an external actor to break the invariants of the state machine and render the cluster unrecoverable.
+const (
+	kubernetesDescriptionKey        = "kubernetes.io/description"
+	kubernetesDescriptionScaryValue = `WARNING: DO NOT EDIT.
```

Comment:
hmm, it would be nice to have immutable secrets, something similar to changing a pod's spec after creation.

Reply:
I think after 4.3 we should consider what other strong protections we can add to prevent cluster destroying mistakes.